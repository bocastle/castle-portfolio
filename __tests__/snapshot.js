import { render } from "@testing-library/react";
import Page from "../src/app/not-found";

it("not-found 페이지 확인::", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
