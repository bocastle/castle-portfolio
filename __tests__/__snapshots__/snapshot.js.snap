// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not-found 페이지 확인:: 1`] = `
<div>
  <div
    class="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 flex flex-col justify-items-center items-center gap-8px"
    role="html"
  >
    <span>
      요청하신 페이지는 진행중입니다.
    </span>
    <span>
      coming soon...
    </span>
  </div>
</div>
`;

exports[`renders homepage unchanged 1`] = `<div />`;
